cdap-app-fabric/src/main/java/co/cask/cdap/config/ConsoleSettingsStore.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.config;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.inject.Inject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+/**
+ * Console Settings Store Management.
+ */
+public class ConsoleSettingsStore {
+  private static final Logger LOG = LoggerFactory.getLogger(ConsoleSettingsStore.class);
+  private static final String NAMESPACE = "";
```

Comment:
Can you use ``CONSOLE_NAMESPACE`` or something more specific (I understand it is a private member of the ``ConsoleSettingsHttpHandler``, though)?